### PR TITLE
Madmax plotting options

### PR DESCRIPTION
**Pull request by graemes**
Added new plot configuration options to madmax (--bucket3 & --rmulti2 ).

(Additional altered log parsing for madmax to mk01's far more elegant version and allows staggering on copy)

### CODE REVIEW

**Comment by guydavis** [comment]
Absolutely beautiful enhancement! Looking forward to seeing these new options available via Plotman in the next release.

File: src/plotman/resources/plotman.yaml [hunk 1]
```diff
@@ -175,3 +175,6 @@ plotting:
                 # executable: /path/to/chia_plot
                 n_threads: 4          # Default is 4, crank up if you have many cores
                 n_buckets: 256        # Default is 256
+                n_buckets3: 256       # Defualt is 256
```
Comment (altendky) [inline src/plotman/resources/plotman.yaml:178]:
```suggestion
                n_buckets3: 256       # Default is 256
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -24,6 +24,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   Allows explicit specification of the plotter executable path if this is preferred over setting the `PATH` environment variable to find the program.
   Presently does not support executables other than the expected names (`chia`, and `chia_plot`).
   ([#823](https://github.com/ericaltendorf/plotman/pull/823))
+- Added new plot configuration options to madmax (--bucket3 & --rmulti2 )
```
Comment (altendky) [inline CHANGELOG.md:27]: This also needs to be moved up into the new unreleased changelog section, but GitHub won't let me suggest up there...

```markdown
- `-v`/`--buckets3` and `-K`/`--rmulti2` are configurable for madMAx.
  ([#869](https://github.com/ericaltendorf/plotman/pull/869))
```

**Review by altendky — CHANGES_REQUESTED** [review]
There are some more general questions about how this fits with other PRs including https://github.com/ericaltendorf/plotman/pull/850 and https://github.com/ericaltendorf/plotman/pull/879.  I have also raised them in Keybase.

File: CHANGELOG.md (hunk 2 above)
Comment (graemes) [reply CHANGELOG.md:27]: Replied on Keybase :-)

**Pull request merged**